### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by tangyuantyy**
Updated 'Displaying item list of all modes' section and Ui.png

### CODE REVIEW

File: docs/images/Ui.png
Comment (jishnu28) [inline docs/images/Ui.png:1]: I was looking through the Ui.png requirements [here](https://nus-cs2103-ay2223s2.github.io/website/admin/tp-deliverables.html#website-ui-png) and I think your new Ui screenshot has some random data and blank spaces for the last 2 columns which the website says we should avoid.. I've taken a new screenshot that should satisfy these requirements, just haven't put up a PR yet, so I can fix this issue!

**Review by jishnu28 — CHANGES_REQUESTED** [review]
Issue with the Ui.png

File: docs/images/Ui.png
Comment (tangyuantyy) [reply docs/images/Ui.png:1]: Ok thank you! I have already reverted the commit

**Review by jishnu28 — APPROVED** [review]
LGTM! Thanks for the correction!

**Pull request merged**